### DISCUSSION TITLE
DataFrame Standardization

### DISCUSSION THREAD

**Discussion by ClariNerd617** [pydata/xarray#8218] [General]
With the ongoing project (currently in draft form) to make an RFC for DataFrames in general, and not just Pandas, are there plans in the roadmap to adopt this generic dataframe object once it is published and formalized?

Link to the current draft of the standard: https://data-apis.org/dataframe-api/draft/

**Comment by max-sixty** [comment]
Not speaking for the project, but quick thoughts:
- I don't think this would fit with the xarray data model, since it looks like the columns can only be 1-dimensional?
- Offering it as an interchange format looks very attractive
